Boolean.CSharp.Main/Account.cs

```diff
@@ -0,0 +1,67 @@
+﻿namespace Boolean.CSharp.Main
+{
+    public abstract class Account
+    {
+        private int Id { get; set; }
+        private Branch Branch { get; set; }
+        public List<Transaction> Transactions { get; private set; }
+
+        public int GetId()
+        {
+            return Id;
+        }
+
+        public Branch GetBranch()
+        {
+            return Branch;
+        }
+
+        protected Account(int id , Branch branch)
+        {
+            Id = id;
+            Branch = branch;
+            Transactions = new List<Transaction>();
+        }
+
+        public bool Deposit(float amount)
+        {
+            if(amount <= 0)
+            {
+                Console.WriteLine("Deposit amount must be greater than zero.");
```

Comment:
not good because if I call this function, I don;t get this info back; you could throw an exception, or give me the error somehow